src/components/menu/components/cart/components/cartItems/CartItems.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
name it as cartItemsList or something like that, sometimes we get confused between this and cartItem